### DISCUSSION TITLE
Customization options for window-theme = auto

### DISCUSSION THREAD

**Discussion by habahabahaba** [ghostty-org/ghostty#3349] [Feature Requests, Ideas] [answered]
As I understand it, if `window-theme = auto`, Ghostty will pick some color from the active theme and modify in some way for the inactive tab background color (I believe, it uses theme background-color for the active tab background). 
Unfortunately, with many dark themes, the contrast between the active and inactive tab backgrounds is very low.

It would be nice if Ghostty could provide a couple of options for generating the background for inactive tabs; 
like, for example, what color of the theme to pick and how to brighten/dim it (e.g. positive number for making it brighter and negative for making it dimmer).

**Comment by 00-kat** [comment, marked as answer]
Duplicate of #3189.
